tools/fuzzer/BUILD

```diff
@@ -106,3 +107,27 @@ cc_test(
         "@com_google_googletest//:gtest_main",
     ],
 )
+
+cc_fuzz_test(
+    name = "lldb_eval_libfuzzer_test",
+    data = [
+        "//testdata:fuzzer_binary_gen",
+        "//testdata:fuzzer_binary_srcs",
+    ],
+    tags = [
+        "manual",
+        # On Linux lldb-server behaves funny in a sandbox ¯\_(ツ)_/¯. This is
+        # not necessary on Windows, but "tags" attribute is not configurable
+        # with select -- https://github.com/bazelbuild/bazel/issues/2971.
+        "no-sandbox",
+    ],
+    srcs = ["lldb_eval_libfuzzer_test.cc"],
+    deps = [
+        "//lldb-eval",
+        "//lldb-eval:runner",
+        "@bazel_tools//tools/cpp/runfiles",
+        ":fuzzer_lib",
```

Comment:
Also needs `"@llvm_project//:lldb-api"`

Reply:
Added. I think this was also missing in `symbol_table_test` and `mutate_test`, so added there too.